### PR TITLE
Adding drupal 9 role and rearch data (PRDS) role

### PR DESCRIPTION
**Pull request by carolyncole**
Allowing PHP to use NGINX as a webserver

@kayiwa would you like a separate PR for the changes to the PHP role?

### CODE REVIEW

File: group_vars/researchdata/researchdata_staging.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+postgres_host: "lib-postgres3.princeton.edu"
```
Comment (kayiwa) [inline group_vars/researchdata/researchdata_staging.yml:1]: no interest in lib-postgres-staging1.princeton.edu?

File: roles/pulibrary.drupal9/molecule/default/tests/test_drupal9.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    assert f.user == 'root'
+    assert f.group == 'root'
+
+
+def test_nginxplus_user(host):
+    user = host.user('www-data')
+
+    assert user.exists
+    assert user.group == 'www-data'
+    assert user.shell == '/usr/sbin/nologin'
```
Comment (kayiwa) [inline roles/pulibrary.drupal9/molecule/default/tests/test_drupal9.py:23]: COOL!

Comment (kayiwa) [inline roles/pulibrary.drupal9/molecule/default/tests/test_drupal9.py:23]: VERY VERY COOL!

File: roles/pulibrary.drupal9/vars/main.yml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+---
+# vars file for roles/pulibrary.drupal9
+nginx_type: opensource
+nginx_install_from: nginx_repository
+php_webserver: 'nginx'
+php_version: '7.4'
```
Comment (kayiwa) [inline roles/pulibrary.drupal9/vars/main.yml:6]: NICE!

File: group_vars/researchdata/researchdata_staging.yml (hunk 1 above)
Comment (carolyncole) [reply group_vars/researchdata/researchdata_staging.yml:1]: @kayiwa I copied from another role and did not try that.  Let me see if it will work...

**Pull request merged**